server.py

```diff
@@ -1,10 +1,16 @@
 from flask import Flask
+from datetime import datetime
+
+now = datetime.now()
```

Comment:
Initializing here will create a time object once and when / is hit again, time is not updated. Please initialize now inside the function `index`

Reply:
The requested changes have been made.